src/main/java/org/fcrepo/importexport/common/BagProfile.java

```diff
@@ -57,6 +72,20 @@ public BagProfile(final InputStream in) throws IOException {
         final ObjectMapper mapper = new ObjectMapper();
         final JsonNode json = mapper.readTree(in);
 
+        loadProfileInfo(json);
+
+        allowFetch = json.has("Allow-Fetch.txt") ? json.get("Allow-Fetch.txt").asBoolean() : true;
```

Comment:
Was debating between having the constants be in the BagProfile class or creating a new class... ended up just creating a BagProfileConstants in 4e3374c since there may be more we will want to add later on.